### ISSUE TITLE
Make titles come first in bundled tiddlers

### ISSUE THREAD

**Issue by twMat** [OokTech/TW5-FederationCore#5]
To manipulate (i.e extract) tiddler data, could bundled tiddlers always start out with the `title : ....` part? As it is, the things-previously-know-as-fields can come in any order.

**Comment by inmysocks** [comment]
I am using the core functions to make the text for the bundles to make sure all of the fields get saved correctly to fix the problem with [[ in fields so the fields block comes in whatever order that returns. Making a new version of that would probably lead to a bunch of other bugs.

This would be simpler if you just said what you are trying to do.

**Comment by twMat** [comment]
The end goal is to give a unified UI for both of FederationCore and TWederBob fetching. For this I want to be able to present "preview cards" and "preview tiddlers" like I did in the TWederBob UI.

In TWederBob there are the preview *tiddlers* and I gave them a template so they can be directly downloaded but the also have a look (e.g background color) and content (from the description and version fields). Also the preview *cards* present this information but not as a tiddler, just a little css thing.

In FedCore bundles I have the titles (in the list field) and it would be doable to extract the tiddlers still in bundle form but, AFAICT, tricky to tell which tiddler belongs to which title unless the title is in a specific position.... ideally as the first "field".... I reckon...
